app/src/main/java/com/kickstarter/ui/viewholders/ProjectViewHolder.java

```diff
@@ -381,6 +387,12 @@ private void setGoalTextView(final @NonNull String goalString) {
     this.goalTextView.setText(goalText);
   }
 
+  private void setProjectActionButtonsVisibility(final Boolean enabled) {
```

Comment:
what's enabled?

Reply:
I renamed it to `isHorizontalRewardsFeatureEnabled`